chrome/manifest.template.js

```diff
@@ -1,46 +1,57 @@
 // @flow
 
+/*::
+type Icons = {
+  '16': string,
+  '48': string,
+  '128': string,
+};
+*/
+
 export default ({
   description,
   defaultTitle,
   contentSecurityPolicy,
   versionName,
-  extensionKey
+  extensionKey,
+  iconOverride,
+  versionOverride,
 } /*: {
   description: string,
   defaultTitle: string,
   contentSecurityPolicy: string,
   versionName?: string,
   extensionKey?: string,
+  iconOverride?: Icons,
+  versionOverride?: string,
 } */
 ) => { // eslint-disable-line function-paren-newline
+  const icons = iconOverride == null
+    ? {
+      /* eslint-disable quote-props */
+      '16': 'img/icon-16.png',
+      '48': 'img/icon-48.png',
+      '128': 'img/icon-128.png',
+      /* eslint-enable quote-props */
+    }
+    : iconOverride;
   const base = {
-    version: '1.10.0',
+    version: versionOverride == null
```

Comment:
I do this so that we can still release a new Byron version as `1.10.0` before Shelley is released if we want